packages/next/client/image.tsx

```diff
@@ -456,22 +457,20 @@ export default function Image({
         srcSet?: string
       }
     | {
+        src: string
         'data-src': string
         'data-srcset'?: string
       }
   if (!lazy) {
     imgAttributes = {
       src: imgSrc,
-    }
-    if (imgSrcSet) {
-      imgAttributes.srcSet = imgSrcSet
+      srcSet: imgSrcSet,
     }
   } else {
     imgAttributes = {
+      src: 'data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=',
```

Comment:
Where did this gif come from? This is the one AMP uses:

```suggestion
      src: 'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACwAAAAAAQABAAACAkQBADs=',
```

Reply:
I can't remember where it came from but there are a lot of suggestions here: https://stackoverflow.com/q/6018611/266535

The current one shows a broke image in Safari so it won't work.

I'm going to use this one which works in Chrome/Safari/Firefox/IE and is transparent:

```
data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7
```